### ISSUE TITLE
C# Samples use Microsoft.WinAppDriver.Appium.WebDriver.1.0.1-Preview

### ISSUE THREAD

**Issue by IhorCo** [microsoft/WinAppDriver#1401]
Should samples be updated to use the latest Appium.WebDriver?

**Comment by liljohnak** [comment]
Someone is trying to update it. https://github.com/microsoft/WinAppDriver/pull/1132

**Comment by IhorCo** [comment]
Yes. But as I can see there only documentation is updated. Would be good to update samples as well. I can prepare a pull request. Does it make sense?

**Comment by naeemakram** [comment]
I can update the samples, I have already used latest version of WinAppDriver with Appium 4 in several projects. I also teach [a course](https://www.udemy.com/course/appium-winappdriver-automation-testing/?referralCode=ED22C3A4CE5BB5E22E53) about it. I'd love to contribute to the project but I don't know where to start.
